SRS.md

```diff
@@ -1,28 +1,98 @@
+## 2. Employees  
+  ### 2.1. Display list of Employees  
+  This mode is intended for viewing and editing the employees list.  
+  
+  __Main scenario:__
+  1. User selects item "Employees";
+  2. Application dispalys list of employees.  
+      
+  ![image](https://user-images.githubusercontent.com/83345134/127285130-2928a7fa-dd6e-4b84-a077-ff80b723aa25.png "Employees list")  
+    Pic 2.1 View the employees list.
+    
+  __The list displays following employees' information:__  
+  1. Department - place, where employee is working;  
+  2. Name of the employee;  
+  3. Date of birth of the employee;  
+  4. Employee's salary.  
+    
+  __Filtering by date:__  
+  1. In the employees list view mode, the user sets a date filter and presses the refresh list button (to the right of the date entry field);  
+  2. The application will show the employees only for a certain period of time.  
 
-![image](https://user-images.githubusercontent.com/83345134/127283891-5c3f5bcd-b593-4d9d-9a74-4706bfdae2cb.png "Departments list")
-![image](https://user-images.githubusercontent.com/83345134/127284946-62fb3d35-98d4-40a6-8d0a-923d58bc3467.png "Add department")
-![image](https://user-images.githubusercontent.com/83345134/127284991-63ca404a-6d7a-400a-a6e1-c7b6b9560a7e.png "Edit department")
-![image](https://user-images.githubusercontent.com/83345134/127285071-2b699f96-0ac5-4421-b7ab-d78414e6d1d0.png "Delete department")
+  __Restrictions:__  
+  * Start date of the period should be less then end date of the period;  
```

Comment:
What happens if the user enters an end date < start date?

Reply:
fised